.postcssrc.js

```diff
@@ -0,0 +1,24 @@
+
+module.exports = {
+	plugins: [
+		require('postcss-prefix-selector')({
```

Comment:
It would be great to have the same rules applied to storybook, so we see changes before installing them in the real Application